generators/docker-compose/templates/_docker-compose.yml

```diff
@@ -14,11 +14,24 @@ services:
             service: kafka
 <%_ } _%>
 <%_ if (gatewayNb + microserviceNb > 0) { _%>
+    <%_ if (serviceDiscoveryType == 'eureka') { _%>
```

Comment:
just in mind, I was told to use === over == everytime i can do...should we then enforce this policy for all of us :smile: ?